.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-/bazel/** @veblush @gnossen
+/bazel/** @veblush @sergiitk
```

Comment:
I don't think you necessarily need to be a Bazel codeowner. This was sort of separate from my Python TL responsibilities. I think the right thing to do here is just to remove me. Feel free to discuss with Esun though.

Reply:
Done.